service/history/service.go

```diff
@@ -107,30 +110,31 @@ func NewConfig(dc *dynamicconfig.Collection, numberOfShards int) *Config {
 		TimerTaskMaxRetryCount:                             5,
 		TimerProcessorGetFailureRetryCount:                 5,
 		TimerProcessorCompleteTimerFailureRetryCount:       10,
-		TimerProcessorUpdateAckInterval:                    10 * time.Second,
-		TimerProcessorForceUpdateInterval:                  10 * time.Minute,
+		TimerProcessorUpdateShardTaskCount:                 100,
```

Comment:
Let's file a task to make these dynamic config knobs.

Reply:
see #592